demo-shell/src/app/components/about/about.component.html

```diff
@@ -1,4 +1,34 @@
 <mat-slide-toggle [(ngModel)]="dev">{{'APP.ABOUT.DEVELOPMENT' | translate }}</mat-slide-toggle>
 
-<adf-about [dev]="dev" [pkg]="pkg"> </adf-about>
+<adf-about>
+    <adf-about-panel *ngIf="dev" [label]="'ABOUT.SERVER_SETTINGS.TITLE' | translate">
+        <ng-template>
+            <adf-about-server-settings></adf-about-server-settings>
+        </ng-template>
+    </adf-about-panel>
+
+    <adf-about-panel *ngIf="dev" [label]="'ABOUT.PACKAGES.TITLE' | translate">
+        <ng-template>
+            <adf-about-package-list [dependencies]="pkg?.dependencies"></adf-about-package-list>
+        </ng-template>
+    </adf-about-panel>
+
+    <adf-about-panel [label]="'ABOUT.REPOSITORY' | translate" *ngIf="repository">
+        <ng-template>
+            <adf-about-repository-info [data]="repository"></adf-about-repository-info>
```

Comment:
I think the name of this component is to tight with

Reply:
yes, in the next refactoring phase, I was planning to move all content related sections to the `content-services` lib